doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -57,30 +64,36 @@ Save this file in the config folder in the robot's moveit_config package with na
 
 * *point_subsample*: Choose one of every *point_subsample* points.
 
-* *padding_offset*: The size of the padding (in cm).
+* *padding_scale*: Should always be >= 1.0. Scale up collision shapes in the scene before excluding them from the octomap.
 
-* *padding_scale*: The scale of the padding.
+* *padding_offset*: Absolute padding (in m) around scaled collision shapes when excluding them from the octomap.
 
 * *filtered_cloud_topic*: The topic on which the filtered cloud will be published (mainly for debugging). The filtering cloud is the resultant cloud after self-filtering has been performed.
 
+* *ns*: An optional namespace for the advertised topics. Required for multiple sensors of the same type.
 
 YAML Configuration file (Depth Map)
 +++++++++++++++++++++++++++++++++++
 
-We will have to generate a YAML configuration file for configuring the 3D sensors. An :panda_codedir:`example file for processing depth images <config/sensors_kinect_depthmap.yaml>` can be found in the panda_moveit_config repository as well.
-Save this file in the config folder in the robot's moveit_config package with name "sensors_kinect_depthmap.yaml": ::
-
- sensors:
-   - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
-     image_topic: /camera/depth_registered/image_raw
-     queue_size: 5
-     near_clipping_plane_distance: 0.3
-     far_clipping_plane_distance: 5.0
-     shadow_threshold: 0.2
-     padding_scale: 4.0
-     padding_offset: 0.03
-     max_update_rate: 1.0
-     filtered_cloud_topic: filtered_cloud
+We will have to generate a YAML configuration file for configuring the 3D sensors. An :panda_codedir:`example file for processing depth images <config/sensors_3d.yaml>` can be found in the panda_moveit_config repository as well.
```

Comment:
Same as above, broken link leading to a non-existing file: https://github.com/ros-planning/panda_moveit_config/blob/rolling-devel/config/sensors_3d.yaml

Reply:
+1. Link updated.